### DISCUSSION TITLE
FYI: Facial rigging example, within ToonSquid

### DISCUSSION THREAD

**Discussion by Cat-Butler** [keiwando/toonsquid#479] [General]
If you missed it being reposted on the [ToonSquid formerly-known-as-Twitter feed](https://twitter.com/toonsquid), [Milkymichii](https://twitter.com/Milkymichii) posted[ an excellent example of facial rigging in ToonSquid](https://twitter.com/Milkymichii/status/1730007504677933296), well worth watching.

**Comment by kevincreative** [comment]
Looks pretty cool! I am new to ToonSquid, is there a tutorial anywhere on how to do this?

**Reply by Cat-Butler** [reply]
Nothing that covers this specifically in detail. Milkymichii did post [a later very quick look at the masking aspect](https://x.com/Milkymichii/status/1731376131784196145).

The [official ToonSquid how-to videos ](https://www.youtube.com/watch?v=bi9oGvfdex4&list=PLWFjpXGMXyoTgIRDrBL38YTUeuvkW4RM7)have [one on using the transform hierarchy for rigging](https://www.youtube.com/watch?v=lWP8ROE5Gwc) and[ one on masking](https://www.youtube.com/watch?v=ZuJIkO1c_08), which explain the features that are used to create this type of rig, but not how they can be combined to create this specific (and impressive) rigging example.

I want to play a bit more with some different body rigs, but I'm hoping to soon put together a few tutorials on rigging (which I prefer to the frame-by-frame drawing approach, although rigging won't work for all contexts), and I'll now include how to do this type of head/face rig (unless anybody does a better version earlier). I'll tag this thread when I start making them public so you get notified.
